package tryCatch;

public class ExceptionTest3 {

	public static void main(String[] args) {  
	System.out.println("접");
		System.out.println("전송");
		System.out.println("해제");

	}

}
